File: 58. Length of Last Word/58. Length of Last Word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        # Remove trailing spaces and split the string into words
        words = s.strip().split()
        
        # If the string is empty after removing spaces, return 0
        if not words:
            return 0
        
        # Return the length of the last word
        return len(words[-1])

File: 58. Length of Last Word/58. Length of Last Word.dart
class Solution {
  int lengthOfLastWord(String s) {
    int length = 0;
    int i = s.length - 1;
    
    // Skip trailing spaces
    while (i >= 0 && s[i] == ' ') {
      i--;
    }
    
    // Count characters of the last word
    while (i >= 0 && s[i] != ' ') {
      length++;
      i--;
    }
    
    return length;
  }
}